pages/bridge.tsx

```diff
@@ -134,12 +224,54 @@ const Bridge = () => {
                 {mounted ? formatUnits(source.fee, 18) : 0} ETH
               </div>
               <div className="text-center">
-                <button
-                  type="button"
-                  className="mt-5 rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
-                >
-                  Bridge to {target.chain.name}
-                </button>
+                {bridgeTarget === BridgeTarget.L2 && needsAllowanceL1 ? (
```

Comment:
Unrelated to this pr, but I noticed the amounts did not round nicely.

![Screenshot from 2023-10-23 22-21-43](https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/12705146/4f842f70-9f53-4019-9714-d0b005ffdd5e)